<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>415</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>415</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>415</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>49</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>50</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="ButtonAdd">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonDelete">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonClear">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonPlay">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Play</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonHelp">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Help</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ButtonOptions">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Options</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_2">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="LabelPosition">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="text">
          <string>Cursor Position:
X:0
Y:0</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="StackedWidget">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="list_actions">
          <layout class="QVBoxLayout" name="verticalLayout">
           <property name="spacing">
            <number>0</number>
           </property>
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QListWidget" name="ListWidget">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="Actions">
          <widget class="QLabel" name="label_2">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>100</y>
             <width>16</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>X:</string>
           </property>
          </widget>
          <widget class="QComboBox" name="ComboBox">
           <property name="geometry">
            <rect>
             <x>40</x>
             <y>50</y>
             <width>171</width>
             <height>22</height>
            </rect>
           </property>
           <item>
            <property name="text">
             <string>Click</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Right Click</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Middle Click</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Double Click</string>
            </property>
           </item>
          </widget>
          <widget class="QLabel" name="label_3">
           <property name="geometry">
            <rect>
             <x>150</x>
             <y>100</y>
             <width>16</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Y:</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="InputWrite">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>240</y>
             <width>231</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
          <widget class="QLineEdit" name="InputPosX">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>90</y>
             <width>41</width>
             <height>20</height>
            </rect>
           </property>
           <property name="inputMethodHints">
            <set>Qt::ImhNone</set>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="maxLength">
            <number>4</number>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
           <property name="placeholderText">
            <string/>
           </property>
          </widget>
          <widget class="QLineEdit" name="InputPosY">
           <property name="geometry">
            <rect>
             <x>170</x>
             <y>90</y>
             <width>41</width>
             <height>20</height>
            </rect>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="maxLength">
            <number>4</number>
           </property>
          </widget>
          <widget class="QCheckBox" name="CheckBox">
           <property name="geometry">
            <rect>
             <x>100</x>
             <y>210</y>
             <width>51</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Write?</string>
           </property>
          </widget>
          <widget class="QSpinBox" name="SpinBoxTimes">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>170</y>
             <width>51</width>
             <height>22</height>
            </rect>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
          <widget class="QLabel" name="label_4">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>170</y>
             <width>31</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Times:</string>
           </property>
          </widget>
          <widget class="QDoubleSpinBox" name="SpinBoxInterval">
           <property name="geometry">
            <rect>
             <x>120</x>
             <y>130</y>
             <width>61</width>
             <height>22</height>
            </rect>
           </property>
           <property name="minimum">
            <double>0.500000000000000</double>
           </property>
           <property name="singleStep">
            <double>0.500000000000000</double>
           </property>
          </widget>
          <widget class="QLabel" name="label_5">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>130</y>
             <width>47</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Interval:</string>
           </property>
          </widget>
          <widget class="QPushButton" name="ButtonOk">
           <property name="geometry">
            <rect>
             <x>180</x>
             <y>300</y>
             <width>61</width>
             <height>51</height>
            </rect>
           </property>
           <property name="text">
            <string>OK</string>
           </property>
          </widget>
          <widget class="QPushButton" name="ButtonCancel">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>300</y>
             <width>61</width>
             <height>51</height>
            </rect>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
          <widget class="QLabel" name="ErrorPos">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>110</y>
             <width>231</width>
             <height>21</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
	color: red;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="ErrorWrite">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>260</y>
             <width>231</width>
             <height>21</height>
            </rect>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
	color: red;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="Options">
          <widget class="QLabel" name="label">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>10</y>
             <width>81</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Options</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QPushButton" name="ButtonOptionsCancel">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>330</y>
             <width>51</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="Help">
          <widget class="QLabel" name="label_6">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>10</y>
             <width>81</width>
             <height>31</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Help</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
          <widget class="QPushButton" name="ButtonHelpCancel">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>330</y>
             <width>51</width>
             <height>20</height>
            </rect>
           </property>
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
          <widget class="QPushButton" name="pushButton_2">
           <property name="geometry">
            <rect>
             <x>190</x>
             <y>330</y>
             <width>51</width>
             <height>21</height>
            </rect>
           </property>
           <property name="text">
            <string>Next</string>
           </property>
          </widget>
          <widget class="QLabel" name="label_7">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>60</y>
             <width>231</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>1 - if you enter an infinite loop move the mouse to one of the corners to stop.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
          <widget class="QLabel" name="label_8">
           <property name="geometry">
            <rect>
             <x>10</x>
             <y>110</y>
             <width>231</width>
             <height>41</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string>2 - Check write whenever you need to  write something after the action.</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
